### DISCUSSION TITLE
Report attribute 'input_data' removed in version 3.0.0

### DISCUSSION THREAD

**Discussion by sirlois** [pytest-dev/pytest-html#460] [Q&A]
Hi,

Until version 2.1.1, report variable contained the attribute 'input_data'. After version 3.0.0 it was removed:

'report' attributes in pytest-html version 2.1.1:
 ['__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__', '__new__', '__re
duce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__test__', '__weakref__', '_from_json', '_get_verbose_word', '_to_json', 'caplog', 'capstderr', 'capstdout', 'count_towards_summary',
 'duration', 'extra', 'failed', 'from_item_and_call', 'fspath', 'get_sections', 'head_line', 'input_data', 'keywords', 'location', 'longrepr', 'longreprtext', 'nodeid', 'outcome', 'passed', 'sections', 'skipped', 'stage_metadata', '
tc_headless', 'toterminal', 'user_properties', 'when']

'report' attributes in pytest-html version 3.0.0:
['__annotations__', '__class__', '__delattr__', '__dict__', '__dir__', '__doc__', '__eq__', '__format__', '__ge__', '__getattribute__', '__gt__', '__hash__', '__init__', '__init_subclass__', '__le__', '__lt__', '__module__', '__ne__
', '__new__', '__reduce__', '__reduce_ex__', '__repr__', '__setattr__', '__sizeof__', '__str__', '__subclasshook__', '__test__', '__weakref__', '_from_json', '_get_verbose_word', '_to_json', 'caplog', 'capstderr', 'capstdout', 'coun
t_towards_summary', 'duration', 'extra', 'failed', 'from_item_and_call', 'fspath', 'get_sections', 'head_line', 'keywords', 'location', 'longrepr', 'longreprtext', 'nodeid', 'outcome', 'passed', 'sections', 'skipped', 'tc_headless',
 'toterminal', 'user_properties', 'when']
 
 Now it is not possible to get the values stored in 'input_data'? How can I get them?
 
Thanks for the help, best regards,
sirlois

**Comment by BeyondEvil** [comment]
I will look at this once v4.0.0 is out.
